gems/sorbet-runtime/test/types/enum.rb

```diff
@@ -121,6 +121,10 @@ class CardSuitCustom < T::Enum
       assert_equal(false, CardSuitCustom.has_serialized?('spade'))
       assert_equal(false, CardSuitCustom.has_serialized?('blerg'))
     end
+
+    it 'does not break for arbitrary objects' do
+      assert_equal(false, CardSuitCustom.has_serialized?(Class.new.new))
```

Comment:
I think what you're trying to do here is check that runtime checking doesn't happen, which is not what this method would do. I don't know if it's worth writing a test for this, versus just a comment next to the `.checked(:never)`  like we do elsewhere.

Reply:
In particular, this test would pass before your change.